core/include/gnuradio-4.0/Graph.hpp

```diff
@@ -343,6 +350,7 @@ class Graph : public gr::Block<Graph> {
         propertyCallbacks[graph::property::kEmplaceEdge]        = std::mem_fn(&Graph::propertyCallbackEmplaceEdge);
         propertyCallbacks[graph::property::kRemoveEdge]         = std::mem_fn(&Graph::propertyCallbackRemoveEdge);
         propertyCallbacks[graph::property::kGraphInspect]       = std::mem_fn(&Graph::propertyCallbackGraphInspect);
+        propertyCallbacks[graph::property::kGraphGRC]           = std::mem_fn(&Graph::propertyCallbackGraphGRC);
```

Comment:
I feel like the semantics of the graph receiving a message and then replacing itself are a bit weird. For my mental model, the scheduler getting that message as the owner of the graph would make a lot more sense?
This might also be that my model of of the interplay between the components is a bit outdated and the graph has become a lot more dynamic lately.